src/java/bobbybot/Task.java

```diff
@@ -0,0 +1,46 @@
+package bobbybot;
+
+/** Represents a task that can be completed
```

Comment:
Should the comment be in the next line after `/**`? Same issue found in `ToDo` class